posix-mapper/src/main/webapp/WEB-INF/web.xml

```diff
@@ -5,8 +5,12 @@
         "http://java.sun.com/j2ee/dtds/web-app_2_3.dtd">
 
 <web-app>
-
     <display-name>posix-mapper</display-name>
+    
+    <listener>
+        <!-- One-time execution to preserve the Identity Manager. -->
```

Comment:
can be removed when listener code moved to InitAction

Reply:
Done.